app/brave_main_delegate.cc

```diff
@@ -93,3 +99,16 @@ void BraveMainDelegate::PreSandboxStartup() {
     brave::InitializeResourceBundle();
   }
 }
+
+bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
+  base::CommandLine& command_line =
+      *base::CommandLine::ForCurrentProcess();
+  command_line.AppendSwitch(switches::kEnableTabAudioMuting);
+
+  std::stringstream enabled_features;
+  enabled_features << features::kEnableEmojiContextMenu.name
+    << "," << features::kDesktopPWAWindowing.name;
```

Comment:
are we sure this is safe to enable? Do you know why it's only enabled by default on Chromeos?

Reply:
should this be flagged for security review @diracdeltas or has someone already looked at it?

Reply:
haven't looked at this. i'm not super familiar with PWAs - i thought they were mainly used on mobile?  i've never heard of a desktop PWA. would prefer to disable unless there's a clear webcompat reason to turn it on.

Reply:
@diracdeltas this would apply to android as well, but afaik chrome also disables them on android

Reply:
As I understand it'll be enabled by default on all platforms in 2018 before we ship in Chrome. It is what chrome apps are being deprecated for.  I am building out a list of things for a security audit, and it will have various things including this.  I would like to keep it enabled since it'd put us in a competitive disadvantage, but we can discuss more at the security audit.

Reply:
well, if it ships in Chrome before we release then won't it switch to enabled by default anyway and we won't need this?